migrator.sh

```diff
@@ -334,7 +334,9 @@ docker_login() {
     # keep retrying docker login until successful
     while [ "$LOGIN_SUCCESS" = "false" ]
     do
-      docker login ${REGISTRY} && LOGIN_SUCCESS="true"
+      read -p "Username: " USERNAME
```

Comment:
I'd prefer if we didn't ask for input when running since we have environment variables that can pass in this information.